zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py

```diff
@@ -18,6 +18,12 @@ class WikipediaHandler(object):
     kind of external issue tracker as well.
     '''
 
+    META = {
+        'name': 'Wikipedia',
+        'description': 'Searches Wikipedia for a term and returns the top article.',
+        'defaults': False,  # Let bot handle all messages
```

Comment:
I think the fact that you feel the need for a comment here suggests we might want a more verbose name than "defaults".  I think "default_commands_enabled" would be a good name.